### DISCUSSION TITLE
Period selection

### DISCUSSION THREAD

**Discussion by ilikemyfto** [alexjustesen/speedtest-tracker#866] [Ideas]
Wish: Is it possible to select a particular month rather than just the last month (or previous 30 days). When its time to discuss my ISP renewal would be helpful to be able to produce data for months where the service was below par for the preceding year.

Currently I have to remember to take a screenshot on the last day of the month.

Otherwise I'm enjoying using the app on my pi4.

**Comment by alexjustesen** [comment]
Attaching this to #599
